### PR TITLE
Polish language translation

### PR DESCRIPTION
**Pull request by jacek-sloma**
Translations for polish language.

### CODE REVIEW

File: flask_user/translations/pl/LC_MESSAGES/flask_user.po [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+#: flask_user/user_manager__views.py:671
+#, python-format
+msgid ""
+"Your email address has not yet been confirmed. Check your email Inbox and"
+" Spam folders for the confirmation email or <a href=\"%(url)s\">Re-send "
+"confirmation email</a>."
+msgstr ""
+"Twój adres email nie został jeszcze potwierdzony. Sprawdź swoją skrzynkę "
+"pocztową, wiadomości przychodzące, jak również folder Spam lub "
+"<a href=\"%(url)s\">Wyślij wiadomość potwierdzającą jeszcze raz."
```
Comment (and-semakin) [inline flask_user/translations/pl/LC_MESSAGES/flask_user.po]: Missing closing `</a>` tag:
```suggestion
"<a href=\"%(url)s\">Wyślij wiadomość potwierdzającą jeszcze raz</a>."
```

Comment (and-semakin) [reply flask_user/translations/pl/LC_MESSAGES/flask_user.po]: Ok, will fix it myself :)

**Comment by and-semakin** [comment]
Thank you again @jacek-sloma!

**Comment by jacek-sloma** [comment]
oh my... :-) I didnt been here for more than a year :-D I will delete this fork and branch.
Best Regards

**Pull request merged**